src/TelegramBase.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace NotificationChannels\Telegram;
+
+use JsonSerializable;
+use NotificationChannels\Telegram\Traits\HasSharedLogic;
+
+/**
+ * Class TelegramMessage.
```

Comment:
Please revise the class name here. Thanks!

Reply:
oh ok done